etc/systemd/system-generators/zfs-mount-generator.in

```diff
@@ -164,13 +165,17 @@ ExecStop=@sbindir@/zfs unload-key '${dataset}'"   > "${dest_norm}/${keyloadunit}
 
   # atime
   if [ "${p_atime}" = on ] ; then
-    # relatime
+    # relatime/lazytime
```

Comment:
This does not appear to match the table from the PR description. For one example, if both `relatime` and `lazytime` are `on`, the table says I should get both mount options, but this only gives me `relatime`.